packages/runtime/test-runtime-utils/src/mockStorage.ts

```diff
@@ -17,41 +17,56 @@ export class MockStorage implements IChannelStorageService {
         return new MockStorage(tree);
     }
 
-    private static readCore(tree: ITree, paths: string[]): string {
+    public async readBlob(path: string): Promise<IBlob> {
+        return this.readBlobInternal(this.tree, path.split("/"));
+    }
+
+    constructor(protected tree?: ITree) {
+    }
+
+    public async read(path: string): Promise<string> {
```

Comment:
As a forcing function, can we remove ALL read() methods from all interfaces?
The goal is to have readBlob() (must) and readString (likely optional, can be even implemented as static function takeing any interface that has readBlob on it) and force all base64 encoding paths out of runtime